mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
```

Comment:
The provided code snippet '})' seems to be the closing of a function or a callback, but it lacks context. It is crucial to ensure that this closing bracket is properly matched with its corresponding opening bracket to avoid syntax errors and maintain the code's functionality. 

Without more context, it's difficult to provide a more specific review. However, I would like to request more code context to ensure that the brackets are properly matched and to identify any potential issues or optimizations.